Hearthstone Deck Tracker/Controls/Overlay/BobsBuddyPanel.xaml.cs

```diff
@@ -287,25 +452,59 @@ private bool CanMinimize
 			=> InCombatPhase && !Config.Instance.ShowBobsBuddyDuringCombat
 			|| InShoppingPhase && !Config.Instance.ShowBobsBuddyDuringShopping;
 
+		public async Task ExpandAverageDamagePanels()
+		{
+			(FindResource("StoryboardExpandAverageDamage") as Storyboard)?.Begin();
+			await Task.Delay(200);
```

Comment:
It doesn't look like this task is awaited anywhere. It's that intended this delay is a no-op.